app/about_us/OurCrew.js

```diff
@@ -0,0 +1,26 @@
+import styles from "./OurCrew.module.css";
+import { ourCrewAbout, ourCrewData } from "./crew.js";
+
+export const OurCrew = () => {
+  return (
+    <section className={styles.container}>
+      <header className={styles.crewHeader}>
+        <p>{ourCrewAbout.description}</p>
+      </header>
+      <div className={styles.crewAbout}>
+        {ourCrewData.map((member, index) => (
+          <div key={index} className={styles.crewItem}>
+            <div className={styles.crewImage}>
+              <img src={member.image} alt={member.name} />
```

Comment:
This `alt="..."` text is not providing any extra info for the non-sighted users, so you can either change it or remove it.
(We are already having the crew members name in the `<h2>` below.)
- More info on alt tag: https://www.w3.org/WAI/tutorials/images/
- Videos about Accessibility testing, and how people use assistive technologies:
  - Short intro video - https://www.youtube.com/watch?v=HE2R86EZPMA
  - Longer and detailed video from a non-sighted user - https://www.youtube.com/watch?v=OUDV1gqs9GA